shacl/register.ttl

```diff
@@ -77,7 +77,6 @@ reg:DistributionShape
 
 reg:OrganizationShape
     a sh:NodeShape ;
-    sh:targetClass schema:Organization ;
```

Comment:
Or there other shapes that we should remove the `targetClass` from?

Reply:
yes, dcat:OrganizationShape
what's left (on purpose): schema:DataCatalog, schema:Dataset, schema:DataDownload and dcat:Dataset, dcat:Distribution